doc/getting-started.rst

```diff
@@ -146,16 +140,16 @@ Recommended for Windows 10
 
 * `Ninja`_ is a useful, fast, build tool, to use it pass ``-GNinja`` to `CMake`_
   the first time it configures a build directory.
-* `clang-format`_ 9, is a tool to format
+* `clang-format`_ is a tool to format
   C/C++/Java/JavaScript/Objective-C/Protobuf code, all oneAPI Construction Kit
   source files are required to be formatted before being committed to the
   repository, including source files which are generated by the build.
 * `lit`_ is a portable tool for executing LLVM and Clang style test suites.
 
 .. code-block:: console
 
-   $ choco install -y ninja llvm doxygen.install
```

Comment:
Not sure on this, but in trying to test it I encountered issues so I've opted to still install it via `choco` but as a separate command with `--version 16.0.6`, so that it should result in having the same version of `clang-format` as in `pip`.